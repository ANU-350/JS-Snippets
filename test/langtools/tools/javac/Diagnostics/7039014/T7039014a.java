/*
 * @test /nodynamiccopyright/
 * @bug     7039014
 * @summary Confusing error message for method conflict
 * @author archiecobbs
 *
 * @compile/fail/ref=T7039014a.out -XDrawDiagnostics T7039014a.java
 */
public class T7039014a {

    interface A<T> {
        byte m(String x);
        char m(T x);
    }

    interface B extends A<String> {
    }
}
