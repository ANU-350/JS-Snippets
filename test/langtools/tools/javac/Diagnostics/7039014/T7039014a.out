T7039014a.java:16:5: compiler.err.incompatible.diff.ret.same.type: T7039014a.A<java.lang.String>, m, java.lang.String
1 error
